designer/client/src/components/history/ProcessHistory.tsx

```diff
@@ -41,13 +41,13 @@ export function ProcessHistoryComponent(props: { isReadOnly?: boolean }): JSX.El
 
     return (
         <Scrollbars
-            renderTrackVertical={(props) => <div {...props} className={styles.innerScroll} />}
+            renderTrackVertical={(props) => <TrackVertical {...props} />}
             renderTrackHorizontal={() => <div className="hide" />}
             autoHeight
             autoHeightMax={300}
             hideTracksWhenNotNeeded={true}
         >
-            <ul id="process-history">
+            <ProcessHistoryWrapper>
```

Comment:
Are you sure we can remove this id `id="process-history"`? maybe it's used as an anchor for some JS events (I didn't see any in the code, but still it's worth to be sure)

Reply:
I didn't find that id at any cases, hm...

Reply:
Ok, so let's leave it